libs/features/cdt/cdt.js

```diff
@@ -0,0 +1,124 @@
+import { getMetadata, getConfig, loadStyle } from '../../utils/utils.js';
+import { replaceKey } from '../placeholders.js';
+
+const replacePlaceholder = async (key) => replaceKey(key, getConfig());
+
+async function loadCountdownTimer(container) {
+  const cdtLabel = await replacePlaceholder('cdt-ends-in');
+  const cdtDays = await replacePlaceholder('cdt-days');
+  const cdtHours = await replacePlaceholder('cdt-hours');
+  const cdtMins = await replacePlaceholder('cdt-mins');
+  const cdtRange = (getMetadata('countdown-timer')).split(',');
+  const timeRangesEpoch = cdtRange.map((time) => Date.parse(time.trim()));
+  let isVisible = false;
+  let interval;
+
+  const { miloLibs, codeRoot } = getConfig();
+  await Promise.resolve(loadStyle(`${miloLibs || codeRoot}/features/cdt/cdt.css`));
+
+  function createTimerFragment(value, label) {
+    const fragment = document.createElement('div');
+    fragment.classList.add('timer-fragment');
+
+    const unitContainer = document.createElement('div');
+    unitContainer.classList.add('timer-unit-container');
+    fragment.appendChild(unitContainer);
+
+    const tensBox = document.createElement('div');
+    tensBox.classList.add('timer-box');
+    tensBox.textContent = Math.floor(value / 10);
+    unitContainer.appendChild(tensBox);
```

Comment:
could we create a small utility in this file that creates element, add class list, text content, and add it to parent ? this would look like
```suggestion
    appendBox('timer-box', Math.floor(value / 10), unitContainer)
```

Reply:
(wonder if you can use utils.createTag in appendBox)